### PR TITLE
feat(textfield): TEDI-Ready TextField development #234

### PR DESCRIPTION
**Pull request by airikej**

### CODE REVIEW

**Comment by kasparsuvi1** [comment]
#234

File: libs/react-components/src/tedi/components/form/textfield/textfield.module.scss [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  cursor: initial;
+}
+
+.tedi-textfield__icon {
+  vertical-align: text-top;
+}
+
+/* Small */
+.tedi-textfield--small {
+  & .tedi-textfield__input {
```
Comment (mart-sessman) [inline libs/react-components/src/tedi/components/form/textfield/textfield.module.scss:161]: Üleliigne tühik?
All pool ka, näiteks rida 183, 188.

**Review by mart-sessman — CHANGES_REQUESTED** [review]

File: libs/react-components/src/tedi/components/form/textfield/textfield.module.scss (hunk 1 above)
Comment (airikej) [reply libs/react-components/src/tedi/components/form/textfield/textfield.module.scss]: This is not a bug and works as if directly nesting child class inside. But i removed the & for clarity

**Review by mart-sessman — APPROVED** [review]

File: libs/react-components/src/community/components/form/select/select.module.scss [hunk 2]
```diff
@@ -77,8 +79,9 @@ div .select__menu {
 
 .select--small {
   & .select__control {
-    min-height: 1.875rem;
-    font-size: 0.75rem;
+    height: $input-height-small;
+    min-height: $input-height-small;
+    font-size: 0.85rem;
```
Comment (kasparsuvi1) [inline libs/react-components/src/community/components/form/select/select.module.scss:84]: This not valid size - 16px * 0.85 = 15,3px.

**Pull request merged**